### ISSUE TITLE
学习率经过多次迭代后变为0

### ISSUE THREAD

**Issue by mrlihellohorld** [PaddlePaddle/PaddleOCR#4045]
配置文件优化器部分：
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
  regularizer:
    name: 'L2'
    factor: 0.00004
训练log中，lr变为0的部分log情况：（batchsize=256）
[2021/09/08 13:16:19] root INFO: epoch: [6/6], iter: 72630, lr: 0.000001, acc: 0.919922, norm_edit_dis: 0.975367, loss: 0.728784, reader_cost: 0.00461 s, batch_cost: 1.90732 s, samples: 2560, ips: 134.21976
[2021/09/08 13:16:58] root INFO: epoch: [6/6], iter: 72640, lr: 0.000001, acc: 0.914062, norm_edit_dis: 0.976232, loss: 0.748411, reader_cost: 0.00474 s, batch_cost: 2.03743 s, samples: 2560, ips: 125.64870
[2021/09/08 13:17:34] root INFO: epoch: [6/6], iter: 72650, lr: 0.000001, acc: 0.921875, norm_edit_dis: 0.977476, loss: 0.624075, reader_cost: 0.00477 s, batch_cost: 1.92778 s, samples: 2560, ips: 132.79515
[2021/09/08 13:18:12] root INFO: epoch: [6/6], iter: 72660, lr: 0.000000, acc: 0.919922, norm_edit_dis: 0.978255, loss: 0.620143, reader_cost: 0.00429 s, batch_cost: 1.90973 s, samples: 2560, ips: 134.05016
[2021/09/08 13:18:50] root INFO: epoch: [6/6], iter: 72670, lr: 0.000000, acc: 0.919922, norm_edit_dis: 0.979986, loss: 0.696734, reader_cost: 0.00434 s, batch_cost: 1.99194 s, samples: 2560, ips: 128.51782
[2021/09/08 13:19:27] root INFO: epoch: [6/6], iter: 72680, lr: 0.000000, acc: 0.917969, norm_edit_dis: 0.974289, loss: 0.711004, reader_cost: 0.00451 s, batch_cost: 1.91017 s, samples: 2560, ips: 134.01935
[2021/09/08 13:20:05] root INFO: epoch: [6/6], iter: 72690, lr: 0.000000, acc: 0.931641, norm_edit_dis: 0.978100, loss: 0.587411, reader_cost: 0.00471 s, batch_cost: 1.90583 s, samples: 2560, ips: 134.32436
[2021/09/08 13:20:43] root INFO: epoch: [6/6], iter: 72700, lr: 0.000000, acc: 0.929688, norm_edit_dis: 0.978754, loss: 0.611883, reader_cost: 0.00456 s, batch_cost: 1.99151 s, samples: 2560, ips: 128.54552
[2021/09/08 13:21:20] root INFO: epoch: [6/6], iter: 72710, lr: 0.000000, acc: 0.921875, norm_edit_dis: 0.974484, loss: 0.713411, reader_cost: 0.00457 s, batch_cost: 1.95270 s, samples: 2560, ips: 131.10041
问题： 我发现lr降为0后，loss和acc都基本没有变化了，但是从曲线来看，如果lr不为0，应该还是可以继续收敛的。请问，如何解决呢

**Comment by tink2123** [comment]
使用cosine衰减，看log应该是训练到了最后一个epoch吧，lr衰减到0了，可以增大迭代轮次。

**Comment by mrlihellohorld** [comment]
谢谢。我知道了。我是6个epoch训练完了，然后接着训练，lr就还是0，所以后面的epoch都没有变化。

**Comment by mrlihellohorld** [comment]
请问我现在训练了10个epoch，学习率已经降为0了，我想再加一些带空格的英文数据集，继续训练，如何修改学习率相关的配置呢。比如，是否应该更换cosine衰减
